Zend/zend_fibers.c

```diff
@@ -622,6 +622,11 @@ ZEND_METHOD(Fiber, this)
 {
 	ZEND_PARSE_PARAMETERS_NONE();
 
+	/* Follow RFC for now */
+	if (EG(current_fiber) == EG(main_fiber)) {
```

Comment:
Even though it seems ridiculous to me, but it's the fast way to solve the BC break here...